service-commands/src/commands/users.js

```diff
@@ -1,13 +1,20 @@
+const axios = require('axios')
 const config = require('../../config/config')
 const fs = require('fs')
 
 const User = {}
+const DISCOVERY_NODE_ENDPOINT = 'http://audius-disc-prov_web-server_1:5000'
```

Comment:
I think this guy is living in two places now?

Reply:
yeahhh, it lives in mad-dog and service-commands. I figure since the two are different repos that we could init this const in these two places?